src/lib/self_monitoring.py

```diff
@@ -297,3 +299,53 @@ def create_self_monitoring_time_series(context: MetricsContext) -> Dict:
     return {"timeSeries": time_series}
 
 
+async def is_self_monitoring_dashboard_exists(context: SfmDashboardsContext, dashboard_display_name: str) -> bool:
+    try:
+        response = await context.gcp_session.request(
+            'GET',
+            url=f"https://monitoring.googleapis.com/v1/projects/{context.project_id_owner}/dashboards",
+            headers={"Authorization": f"Bearer {context.token}"}
+        )
+        if response.status <= 202:
+            response_json = await response.json()
+            return dashboard_display_name in [dashboard.get("displayName") for dashboard in response_json.get("dashboards")]
+    except Exception as e:
+        context.log(f"Failed to list self monitoring dashboards, because: {e}")
+    return False
+
+
+async def create_new_dashboard(context: SfmDashboardsContext, dashboard: Dict):
+    response = await context.gcp_session.request(
+        "POST",
+        url=f"https://monitoring.googleapis.com/v1/projects/{context.project_id_owner}/dashboards",
+        data=json.dumps(dashboard),
+        headers={"Authorization": f"Bearer {context.token}"}
+    )
+
+    if response.status > 202:
+        response_body = await response.json()
+        context.log(f"Failed to create self monitoring dashboard for due to '{response_body}'")
+    else:
+        context.log(f"The self monitoring dashboard '{dashboard.get('displayName')}' correctly imported")
+
+
+async def import_self_monitoring_dashboard(context: SfmDashboardsContext):
+    if context.operation_mode == OperationMode.Metrics:
+        dashboard_filename = "dynatrace-gcp-function_self_monitoring.json"
+    elif context.operation_mode == OperationMode.Logs:
+        dashboard_filename = "dynatrace-gcp-function-log-self-monitoring.json"
+    else:
+        context.log(f"Lack of self monitoring dashboard for '{context.operation_mode}' operation mode")
+        return
+
+    dashboard_file_path = os.path.dirname(os.path.realpath(__file__)).join("../config").join(dashboard_filename)
```

Comment:
Shouldn't the dashboard file be taken from **dashboard** folder instead of **config**?